package.json

```diff
@@ -40,6 +40,7 @@
       "<rootDir>/{components,packages}/*/src/**/?(*.)(spec|test).js?(x)"
     ],
     "testEnvironment": "node",
+    "testURL": "http://localhost",
```

Comment:
why has this been added?

this seems like a superfluous change - i don't understand what it's doing, and as it's not needed we should remove

Reply:
ah ok - as it's a jest config thing only that's cool :grin: